scripts/build_release.sh

```diff
@@ -132,18 +141,23 @@ allow-preset-passphrase
 default-cache-ttl 2592000
 max-cache-ttl 2592000
 EOF
-gpg --generate-key --batch ${HOME}/tkey/keygenscript
-gpg --export -a > "${HOME}/docker_test_resources/key.pub"
+gpg --gen-key --batch ${HOME}/tkey/keygenscript
+gpg --gen-key --batch ${HOME}/tkey/rpmkeygenscript
+gpg --export -a dev@algorand.com > "${HOME}/docker_test_resources/key.pub"
+gpg --export -a rpm@algorand.com > "${HOME}/docker_test_resources/rpm.pub"
```

Comment:
Why two different keys?

Reply:
dev@ has better security properties and we use it to sign a bunch of things .tar.gz and .deb
rpm is dumb and requires a simpler gpg key so it gets a simpler lesser key.